Framework/Algorithms/inc/MantidAlgorithms/Stitch1DMany.h

```diff
@@ -94,13 +93,13 @@ class DLLExport Stitch1DMany : public API::Algorithm {
   std::vector<double> m_endOverlaps;
   std::vector<double> m_params;
   std::vector<double> m_scaleFactors;
+  std::vector<double> m_manualScaleFactors;
```

Comment:
Could we tweak the parameter name to `scaleFactors` so other developers don't fall into the same trap?